### PR TITLE
Update dotnet message-port guides

### PR DESCRIPTION
**Pull request by kilig**
Update dotnet message-port guides

### CODE REVIEW

**Comment by surbhig12** [comment]
Hello @sabeena21, request you to review his file.

**Comment by sabeena21** [comment]
Hello,

As discussed, I am attaching the reviewed file.
Please find the path of the reviewed file: 
https://github.com/Samsung/tizen-docs/files/1806483/message-port.zip.

Regards,
Sabeena

**Comment by mijin85cho** [comment]
Please don't use the attached file yet. This file should be updated.

File: docs/application/dotnet/guides/app-management/message-port.md [hunk 1]
```diff
@@ -104,31 +102,31 @@ To send a message from one application (`LocPortApp.Tizen`) to another (`RmtPort
...
+        Log.Debug(TAG, "Message Received");
+        Log.Debug(TAG, "App ID: " + e.Remote.AppId);
+        Log.Debug(TAG, "PortName: " + e.Remote.PortName);
+        Log.Debug(TAG, "Trusted: " + e.Remote.Trusted);
+        Log.Debug(TAG, "message: " + e.Message.GetItem <string> ("message"));
     }
     ```
 
-3.  In the sending application, send the message with the `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, providing the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class:
+3.  In the sending application, to send the message should register a local port, call the `Listen()` method of the `Tizen.Applications.Messages.MessagePort` class. And send the message with the `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, providing the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class:
```
Comment (sabeena21) [inline docs/application/dotnet/guides/app-management/message-port.md]: Can be rephrased as below:

In the sending application, to send the message, follow the steps:

	a.  You must register to a local port.
	
	b.  Call the `Listen()` method `Tizen.Applications.Messages.MessagePort` class.
	
	c.  Use `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, to send the message.
	
	d.  Provide the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class as shown in the below code:

File: docs/application/dotnet/guides/app-management/message-port.md [hunk 2]
```diff
@@ -104,31 +102,35 @@ To send a message from one application (`LocPortApp.Tizen`) to another (`RmtPort
...
+        Log.Debug(TAG, "App ID: " + e.Remote.AppId);
+        Log.Debug(TAG, "PortName: " + e.Remote.PortName);
+        Log.Debug(TAG, "Trusted: " + e.Remote.Trusted);
+        Log.Debug(TAG, "message: " + e.Message.GetItem <string> ("message"));
     }
     ```
 
-3.  In the sending application, send the message with the `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, providing the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class:
+3.  In the sending application, to send the message, follow the steps:
+    a. You must register to a local port.
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/app-management/message-port.md]: Please insert empty lines like below to make line breaks. MD consider empty lines as a kind of code also.
```
	a.  You must register to a local port.
(empty line)
	b.  Call the `Listen()` method `Tizen.Applications.Messages.MessagePort` class.
(empty line)
	c.  Use `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, to send the message.
(empty line)
	d. Provide the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class as shown in the below code:
(empty line)
```

**Comment by mijin85cho** [comment]
Thanks for the update kilig. 
Sabeena, if thing are fine, please approve this PR with "Review Changes". When this PR is approved, I'll merge it.

**Comment by mijin85cho** [comment]
Thanks for the check. : )

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**